cli/plugins/go-deps/post_bazel.py

```diff
@@ -26,7 +26,7 @@ def main():
             os.environ.get("BUILD_WORKSPACE_DIRECTORY", "")
         )
         m = re.search(
-            f'^\s*(?P<prefix>.*/execroot/|/tmp/bazel-working-directory/)(({workspace_basename}|_main)/)?(?P<src_path>.*?\.go): import of "(?P<import_url>.*?)"',
+            f'^\\s*(?P<prefix>.*/execroot/|/tmp/bazel-working-directory/)(({workspace_basename}|_main)/)?(?P<src_path>.*?\\.go): import of "(?P<import_url>.*?)"',
```

Comment:
nit: use raw string to avoid having to escape

```suggestion
            rf'^\s*(?P<prefix>.*/execroot/|/tmp/bazel-working-directory/)(({workspace_basename}|_main)/)?(?P<src_path>.*?\.go): import of "(?P<import_url>.*?)"',
```